docs/source/wakepy-mode-lifecycle.md

```
@@ -47,6 +52,26 @@ def long_running_task():
 The rest of this document uses the context manager syntax in the examples because it makes the Mode lifecycle steps more explicit. The decorator syntax is just some syntactic sugar and uses a context manager internally automatically every time the decorated function is called.
 ```
 
+### Wakepy Mode Activation on High Level
+
+The diagram below illustrates how wakepy selects and prioritizes *Methods*, then activates a *Mode*.
+
+1. **Selected Methods**: When a {class}`~wakepy.Mode` instance is created, all Methods that support the Mode are filtered using the optional `omit` or `methods` input arguments to form the *Selected Methods* list.
+2. **WakepyFakeSuccess**: The [`WAKEPY_FAKE_SUCCESS`](#WAKEPY_FAKE_SUCCESS) option may add an additional WakepyFakeSuccess Method to the beginning of the list.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)